buddyboss/buddyboss-platform
# #641 - Repair and import tools are inaccessible when the platform is network activated

Repair and import tools are inaccessible when the platform is network activated issue fixed

Fixed the Repair Community menu for multisite

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #641 .

### How to test the changes in this Pull Request:

1. In Multisite environment, network activate BuddyBoss Platform
2. Go to Network admin dashboard > BuddyBoss > Tools > Repair Community, you will notice that it is not properly displaying.
3. Go to Network admin dashboard > BuddyBoss > Tools, and try to navigate on the following tools:

- Repair Forums
- Import Profile Types
- import Forums
- Import Media

### Proof Screenshots or Video



### Other information:
